draft-packages/filter-menu-button/KaizenDraft/FilterMenuButton/FilterSplitButton.module.scss

```diff
@@ -119,6 +119,11 @@
   letter-spacing: $kz-var-typography-button-secondary-letter-spacing;
 }
 
+// Account for border width affecting overall button size
+.buttonNotActive {
```

Comment:
Could we perhaps always have a border? We can make it transparent when it's not active and add the active color when it is?

Reply:
Yeah that's a cleaner solution, I've pushed up that change now